__tests__/app.test.js

```diff
@@ -376,3 +376,30 @@ describe('GET /api/users', () => {
       });
   });
 });
+
+describe('DELETE /api/comments/:comment_id', () => {
+  it('204: deletes the comment specified by comment_id', () => {
+    return request(app)
+      .delete('/api/comments/1')
+      .expect(204)
+      .then(({ body }) => {
+        expect(body.rows).toBe(undefined);
```

Comment:
this assertion can be removed, express will _never_ send a body if your status is 204